dnd-character-generator/src/components/CharacterInfo.jsx

```diff
@@ -1,12 +1,12 @@
 export function CharacterInfo({ charClass, race, background, alignment }) {
     return (
       <div className="bg-[#e1d3b8] border p-4 grid grid-cols-2 gap-2 text-sm">
-        <div>Class & Level:</div><div>{charClass}</div>
-        <div>Race:</div><div>{race}</div>
-        <div>Background:</div><div>{background}</div>
-        <div>Alignment:</div><div>{alignment}</div>
-        <div>Experience Points:</div><div><input type="text" className="border px-1 w-full" /></div>
-        <div>Player Name:</div><div><input type="text" className="border px-1 w-full" /></div>
+        <div className="font-bold">Class & Level:</div><div>{charClass}</div>
+        <div className="font-bold">Race:</div><div>{race}</div>
+        <div className="font-bold">Background:</div><div>{background}</div>
+        <div className="font-bold">Alignment:</div><div>{alignment}</div>
+        <div className="font-bold">Experience Points:</div><div><input type="text" placeholder="Enter Experience points" className="bg-white border px-1 w-full" /></div>
+        <div className="font-bold">Player Name:</div><div><input type="text" placeholder="Enter your name" className="bg-white border px-1 w-full" /></div>
```

Comment:
Divjes zijn eigenlijk niet voor tekst. Ik zou hier strongs en spannetjes voor gebruiken. Door de string is de font-bold ook niet meer nodig

```suggestion
      <strong>Class & Level:</strong>
      <span>{charClass}</span>
      <strong>Race:</strong>
      <span>{race}</span>
      <strong>Background:</strong>
      <span>{background}</span>
      <strong>Alignment:</strong>
      <span>{alignment}</span>
      <strong>Experience Points:</strong>
      <span>
        <input
          type="text"
          placeholder="Enter Experience points"
          className="bg-white border px-1 w-full"
        />
      </span>
      <strong>Player Name:</strong>
      <span>
        <input
          type="text"
          placeholder="Enter your name"
          className="bg-white border px-1 w-full"
        />
      </span>
```